static/_redirects

```diff
@@ -33,6 +33,9 @@
 /calico-enterprise/3.15/ /calico-enterprise/3.15/about-calico-enterprise 301
 /calico-enterprise/3.14/ /calico-enterprise/3.15/about-calico-enterprise 301
 
+# Redirects for CC pages
+/calico-cloud/latest/image-assurance/scanners/cli-based-scanner#start-the-cli-scanner /calico-cloud/latest/image-assurance/overview 301
```

Comment:
Hi @bmckercher123, I think we're still not on the same page. 

If I understand the syntax in this redirect file it is: 
```
source_url dest_url 
```

If my understanding above is correct, then the target URL you have is: 
- /calico-cloud/latest/image-assurance/overview

That target is not correct. 

We want the target (destination) URL to be: 
- /calico-cloud/image-assurance/scanners/cli-based-scanner#start-the-cli-scanner

Let me know if that doesn't make sense to you. 

Also, I noticed you have `/latest` in the URLs. I tried hitting `https://docs.tigera.io/calico-cloud/latest/image-assurance/overview` and it gives me a 404.